app/javascript/Search/Search.jsx

```diff
@@ -109,7 +109,8 @@ export class Search extends Component {
   componentWillUnmount() {
     document.removeEventListener('keydown', this.globalKeysListener);
     window.removeEventListener('popstate', this.syncSearchUrlWithInput);
-    InstantClick.off('change', this.enableSearchPageListener);
+    InstantClick.off &&
```

Comment:
This addition existence check is required due to the component being explicitly unmounted outside the normal Preact component lifecycle.

Reply:
Would this comment make sense as a source code comment?

Reply:
Done.